src/python/pants/base/exception_sink.py

```diff
@@ -188,6 +192,16 @@ def reset_log_location(cls, new_log_location: str) -> None:
         cls._pid_specific_error_fileobj = pid_specific_error_stream
         cls._shared_error_fileobj = shared_error_stream
 
+    @classmethod
+    def set_logging_initialized(cls):
+        """Set the flag (to True) which indicates that the rust logger has been initialized.
+
+        Class state:
```

Comment:
I don't think it's useful to have these `Class state` messages in the docstrings for simple class accessor functions like this. It adds a lot of additional verbosity for relatively little information.